docs/agent.rst

```diff
@@ -48,54 +48,50 @@ Run an Agent
-For Serial communication (only Linux): ::
+**TCP communication** ::
 
-    $ ./MicroXRCEAgent serial <device> [<baudrate>]
+    $ ./MicroXRCEAgent --tcp <local-port> [--discovery <discovery-port>] [--refs <refs-file>]
 
-The ``<device>`` is the device file that represents the serial port, and the ``<baudrate>`` is the baud rate of the communication which can take the following values:
-0, 50, 75, 110, 134, 150, 200, 300, 600, 1200, 1800, 240, 4800, 9600, 19200, 38400, 57600, 115200 (default), 230400, 460800, 500000, 576000, 921600, 1000000, 1152000, 1500000, 2000000, 2500000, 3000000, 3500000 or 4000000 Bd;
-any other value will be converted to ``int`` using the ``atoi`` function.
+Launch a TCP server over port ``<local-port``. It has the following options:
 
-For Pseudo-Serial communication (only Linux): ::
+* ``--discovery <discovery-port>`` (only Linux): select a port for the discovery server which has the port number ``<7400>`` by default.
+* ``--refs <refs-file>``: load a reference file using the relative path to the working directory.
+  This reference file shall be composed by a set of Fast RTPS profile following the `XML syntax <https://eprosima-fast-rtps.readthedocs.io/en/latest/xmlprofiles.html>`_ describes in Fast RTPS.
+  The ``profile_name`` attribute of each profile represents a reference for an XRCE-Entity, so it could be used by the *Clients* in order to create entities by reference.
 
-    $ ./MicroXRCEAgent pseudo-serial [<baudrate>]
+**Serial communication** (only Linux) ::
 
-In this case, a pseudo-terminal will be created in order to simulate a serial communication.
-As in the serial transport, the baud rate could be set using the ``<baudrate>`` parameter.
+    $ ./MicroXRCEAgent --serial <device-name> [--baudrate <baudrate>] [--refs <refs-file>]
 
-If the transport used is a reliable transport, the use of a best-effort stream over it is equivalent to use a reliable stream over not reliable transport.
+Launch a Serial server over device ``<device-name>>``. It has the following options:
 
-If the transport used is a reliable transport, the use of a best-effort stream over it is equivalent to use a reliable stream over a not reliable transport.
+* ``--baudrate <baudrate>``: set the baud rate of the communication. It can take the following values:
+  0, 50, 75, 110, 134, 150, 200, 300, 600, 1200, 1800, 240, 4800, 9600, 19200, 38400, 57600, 115200 (default), 230400, 460800, 500000, 576000, 921600, 1000000, 1152000, 1500000, 2000000, 2500000, 3000000, 3500000 or 4000000 Bd.
+* ``--refs <refs-file>``: load a reference file using the relative path to the working directory.
+  This reference file shall be composed by a set of Fast RTPS profile following the `XML syntax <https://eprosima-fast-rtps.readthedocs.io/en/latest/xmlprofiles.html>`_ describes in Fast RTPS.
+  The ``profile_name`` attribute of each profile represents a reference for an XRCE-Entity, so it could be used by the *Clients* in order to create entities by reference.
```

Comment:
```suggestion
  The ``profile_name`` attribute of each profile represents a reference for an XRCE-Entity so that it could be used by the *Clients* to create entities by reference.
```

Reply:
Done!